### PR TITLE
chore(template): add  Regular Tasks for Feature Release template

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue longhorn#9682

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/regular-tasks-for-release.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Regular tasks for feature release v<x.y>.0
+
+## Describe the sub-tasks
+
+- [ ] OS Distro Version Update (QA captain)
+  - [ ] Verify by ci.longhorn.io/job/public/job
+  - [ ] Update `Best Practices>Operating System` in the official document
+ 
+- [ ] BCI Image Update
```
Comment (innobead) [inline .github/ISSUE_TEMPLATE/regular-tasks-for-release.md:19]:
```suggestion
- [ ] BCI Image Update for Component Container Base Image
```

File: .github/ISSUE_TEMPLATE/regular-tasks-for-release.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+name: Regular Tasks for Feature Release
+about: Regular tasks for a feature release
+title: "[RELEASE] Regular Tasks for Feature Release v<x.y>.0"
+labels: release/task
+assignees: ''
+---
+
+## What's the task? Please describe
+
+Regular tasks for feature release v<x.y>.0
```
Comment (innobead) [inline .github/ISSUE_TEMPLATE/regular-tasks-for-release.md:11]:
```suggestion
Regular tasks for feature release v<x.y>.0 must be completed well before the release, during the development phase.

For OS distro updates for SLES and SLE Micro, need to update for each patch release as well. 
```

File: .github/ISSUE_TEMPLATE/regular-tasks-for-release.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Describe the sub-tasks
+
+- [ ] OS Distro Version Update (QA captain)
```
Comment (innobead) [inline .github/ISSUE_TEMPLATE/regular-tasks-for-release.md:15]: Add another setting for QA.

- [ ] K8s Distro Version Update for Testing Pipelines (QA captain)
  - [ ] Update to the latest supported K8s version (RKE2, K3s)

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just a few suggestions.

**Comment by innobead** [comment]
BTW, for execution, we should do this earlier instead of waiting until just before the release, probably in the middle of the release development.

**Comment by derekbit** [comment]
All updated. Thank you.

File: .github/ISSUE_TEMPLATE/regular-tasks-for-release.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Regular tasks for feature release v<x.y>.0 must be completed well before the release, during the development phase.
+
+For OS distro updates for SLES and SLE Micro, need to update for each patch release as well. 
+
+## Describe the sub-tasks
+
+- [ ] OS Distro Version Update (QA captain)
+  - [ ] Verify by ci.longhorn.io/job/public/job
+  - [ ] Update `Best Practices>Operating System` in the official document
```
Comment (innobead) [inline .github/ISSUE_TEMPLATE/regular-tasks-for-release.md:19]:
```suggestion
  - [ ] Update `Best Practices>Operating System` in the official document and testing pipelines
```

File: .github/ISSUE_TEMPLATE/regular-tasks-for-release.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- [ ] K8s Distro Version Update for Testing Pipelines (QA captain)
+  - [ ]Update to the latest supported K8s version (RKE2, K3s)
```
Comment (innobead) [inline .github/ISSUE_TEMPLATE/regular-tasks-for-release.md:22]:
```suggestion
  - [ ] Update the testing pipelines to use the latest supported K8s version (RKE2, K3s)
```

File: .github/ISSUE_TEMPLATE/regular-tasks-for-release.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+For OS distro updates for SLES and SLE Micro, need to update for each patch release as well. 
+
+## Describe the sub-tasks
+
+- [ ] OS Distro Version Update (QA captain)
+  - [ ] Verify by ci.longhorn.io/job/public/job
+  - [ ] Update `Best Practices>Operating System` in the official document
+
+- [ ] K8s Distro Version Update for Testing Pipelines (QA captain)
```
Comment (innobead) [inline .github/ISSUE_TEMPLATE/regular-tasks-for-release.md:21]:
```suggestion
- [ ] K8s Distro Version Update (QA captain)
```

**Review by innobead — APPROVED** [review]

**Pull request merged**